### DISCUSSION TITLE
FeatureProposal: Allow developers to customize the threshold trigger of SwipeControl

### DISCUSSION THREAD

**Discussion by Psykobal** [microsoft/microsoft-ui-xaml#7494] [Q&A]
I find the SwipeItems in Execute mode too sensitive, the users often executed it by mistake. I would like to change the threshold to make it harder to invoke. 
In the [docs](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/swipe#how-does-swipe-work) it says: "_If the user swipes past the threshold and then releases the item, the command is executed immediately._". ~~Is~~ **can we make it** it possible to change it?

**Comment by StephenLPeters** [comment]
looking at the code unfortunately I don't think there is currently. https://github.com/microsoft/microsoft-ui-xaml/blob/cb19f34a15115f889ba1854d2ed006792e9d0982/dev/SwipeControl/SwipeControl.cpp#L18  This is the value which controls the threshold, its just a static in the cpp file. I think it makes sense to expose this as a resource so that it can be overwritten and customized. :) @kikisaints What do you think?
